test.py

```diff
@@ -0,0 +1,595 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+import argparse
+import logging
+import shutil
+import subprocess
+import tempfile
+import time
+
+def get_repo_root():
+    return os.path.dirname(sys.argv[0])
+
+logger = logging.getLogger("dcos-commons-test")
+logging.basicConfig(level=logging.INFO, format="%(asctime)s %(name)s %(message)s")
+
+sys.path.append(os.path.join(get_repo_root(), 'tools'))
```

Comment:
the bulk of test.py probably doesn't belong in the root.  I'm not racing to move the location of every .py file in tools though, and that's why i did the path monkeying.